src/lib.rs

```diff
@@ -84,3 +106,44 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     let polygon = Polygon::new(LineString::from(circle_points), vec![]);
     polygon
 }
+
+// Make a single clock polygon
+pub fn clockpoly(
+    centerpoint: Point<f64>,
+    radius: f64,
+    inner_radius: f64,
+    num_vertices: usize,
+    num_segments: usize,
+    seg: usize,
+) -> Polygon<f64> {
+    let mut circle_points = Vec::new();
```

Comment:
Likewise, how about `circle_points_outer`? Or since this isn't a full circle, even `outer_arc` and `inner_arc`?

Reply:
Updated.